packages/tools/src/expandPackageGlobs.ts

```diff
@@ -26,19 +26,21 @@ export async function expandPackageGlobs(
   const discoveredPackages: Array<Package | undefined> = await Promise.all(
     directories.map((dir) =>
       fs
-        .readJson(path.join(dir, "package.json"))
+        .promises
```

Comment:
rogue `fs.promises` here :P

Reply:
In general, this thing here could be rewritten to make use of the util that u introduced. But I get that it's not really the focus of this PR so I'm fine leaving it out from here.

Reply:
Ah, thanks. I missed this one with my search since it was split into two lines. I've just left it written in the existing style for now to avoid changing too much and avoid introducing any accidental changes